src/utils.ts

```diff
@@ -0,0 +1,8 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import path from 'node:path';
```

Comment:
There is `path.posix` export to get that version regardless the current platform. Also there is [pathe](https://www.npmjs.com/package/pathe) 3rd party module, which you may find useful in this task